main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/codescalersinternships/Datetime-Client-MohamedFadel/client"
+)
+
+func main() {
+	serverURL := os.Getenv("SERVER_URL")
+	serverPort := os.Getenv("SERVER_PORT")
+	serverPortGin := os.Getenv("SERVER_PORT_GIN")
```

Comment:
I don't think this should be done, the client should fetch data from whatever server, so no need to force the server to fetch data from both servers every time